### ISSUE TITLE
can't pass variable to sealed test env

### ISSUE THREAD

**Issue by arthurep** [oknozor/sealed-test#6]
I would to run a sealed test with a custom env (before running the process). However I cannot pass a variable to the env attribute. It seems it only accepts string literals. Not even static variables work:

```
const SOME_STR: &str = "hello";

#[sealed_test(env = [ ("SOME_VAR", SOME_STR) ])]
fn my_test() -> Result<()> {
    Ok(())
}
```
I get the error `unexpected token, expected `)``.

**Comment by oknozor** [comment]
indeed, this need to be updated, meanwhile you can just `std::env::set_var` at the start of the test.

**Comment by arthurep** [comment]
thanks for the reply. Unfortunately this won't work for my usecase, since I need to set LD_LIBRARY_PATH before the new process is forked, in order to run the test using a different library path. The only way would be to have this set before the work.

**Comment by oknozor** [comment]
have you tried using the `before` arg in the macro attribute ?
